docs/built-in-functions.rst

```diff
@@ -379,7 +379,11 @@ Cryptography
     * ``s``: second 32 bytes of signature
     * ``v``: final 1 byte of signature
 
-    Returns the associated address, or ``0`` on error.
+    Returns the associated address, or ``empty(address)`` on error.
```

Comment:
wait a second, it shouldn't return anything on error right?

Reply:
That's the current behaviour which doesn't revert. Iiuc we haven't implemented a revert yet for an error?

Reply:
Yes confirmed, here the corresponding test: https://github.com/vyperlang/vyper/commit/019a37ab98ff53f04fecfadf602b6cd5ac748f7f#diff-5ae2c19276aba784186401cd67ae4a7c50a67cf2b0c30a96dfe1d2037ac4d570R60